pycvvdp/video_source_file.py

```diff
@@ -70,7 +70,10 @@ class video_reader:
 
     def __init__(self, vidfile, frames=-1, resize_fn=None, resize_height=-1, resize_width=-1, verbose=False):
         try:
-            probe = ffmpeg.probe(vidfile)
+            if vidfile.lower().endswith('.y4m'):
+                probe = ffmpeg.probe(vidfile, count_frames=None)
```

Comment:
We could simply have `count_frames` by default right, so if there are any other ingest supported by ffmpeg and not y4m which do not have frame_count by default, would the software support it? The overhead wouldn't be much and would be cleaner?

Reply:
I'm not opposed to doing it by default, the overhead of counting frames is a rounding error compared to running ColorVideoVDP. 

I just wanted to make the change as minimally invasive as needed to solve the problem, but happy to adjust.